flexmeasures/data/models/planning/tests/test_solver.py

```diff
@@ -188,6 +167,90 @@ def test_battery_solver_day_2(
         )
 
 
+@pytest.mark.parametrize("use_inflexible_device", [False, True])
+def test_battery_solver_day_3(
+    # add_battery_assets, add_inflexible_device_forecasts, use_inflexible_device
```

Comment:
This line can go. But this actually also still needs some thinking about why we want to check 2 cases: with and without an inflexible device. Do we expect to see a difference in the results? If anything, it's a todo, because I haven't thought about it yet.